docs/parsers/RSSParser.md

```diff
@@ -0,0 +1,30 @@
+---
+title: RSS Parser
+author: williamabradley
+ms.date: 01/19/2018
+description: The RSS Parser allows you to parse an RSS content String into RSS Schema.
+keywords: uwp community toolkit, uwp toolkit, microsoft community toolkit, microsoft toolkit, rss, rss parsing, parser
+---
+
+# RSS Parser
+
+The [RssParser](https://docs.microsoft.com/en-us/dotnet/api/microsoft.toolkit.parsers.rss.rssparser) class allows you to parse a RSS content String into a RSS Schema.
+
+| Class | Purpose |
```

Comment:
Properties should be in a section called Properties (see template: https://github.com/Microsoft/UWPCommunityToolkit/blob/master/docs/.template.md)